RecoLocalTracker/Phase2TrackerRecHits/src/Phase2StripCPEGeometric.cc

```diff
@@ -0,0 +1,28 @@
+
+#include "RecoLocalTracker/Phase2TrackerRecHits/interface/Phase2StripCPEGeometric.h"
+#include "SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerFwd.h"
```

Comment:
@slava77 , it is available for IBs. We are not running it for PR tests yet.
@iahmad-khan , can you please include it for pr tests?